tests/zfs-tests/cmd/xattrtest/xattrtest.c

```diff
@@ -98,11 +98,12 @@ static char script[PATH_MAX] = "/bin/true";
 static char xattrbytes[XATTR_SIZE_MAX];
 
 static int
-usage(int argc, char **argv) {
+usage(int argc, char **argv)
+{
 	fprintf(stderr,
-	"usage: %s [-hvycdrRk] [-n <nth>] [-f <files>] [-x <xattrs>]\n"
-	"       [-s <bytes>] [-p <path>] [-t <script> ] [-o <phase>]\n",
-	argv[0]);
+	    "usage: %s [-hvycdrRk] [-n <nth>] [-f <files>] [-x <xattrs>]\n"
+	    "       [-s <bytes>] [-p <path>] [-t <script> ] [-o <phase>]\n",
+	    argv[0]);
 
 	fprintf(stderr,
```

Comment:
It's interesting, i formatted it.
There are some OpenZFS commits related to it, i'll port them after this PR.